### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by jeffyjf**
**What this PR does / why we need it**:

I found some typos by vsocde [spellchecker plugin](https://github.com/streetsidesoftware/vscode-spell-checker).

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jeffyjf** [comment]
> This is great, thank you so much @jeffyjf! I'll leave setting the other label to another member of the team, I'd like another opinion on changing user-facing messages.

In my option, they should be corrected. Due to some log system or monitor system maybe use regular expression to filter the results, in this case incorrect spelling will result in incorrect result.

**Comment by jeffyjf** [comment]
@jichenjc @kayrus @zetaab asking for review
Thanks in advnace

File: pkg/csi/cinder/openstack/openstack_snapshots.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func (os *OpenStack) WaitSnapshotReady(snapshotID string) error {
 	})
 
 	if wait.Interrupted(err) {
-		err = fmt.Errorf("Timeout, Snapshot  %s is still not Ready %v", snapshotID, err)
```
Comment (kayrus) [inline pkg/csi/cinder/openstack/openstack_snapshots.go:176]: I wonder why don't we just return here

**Review by kayrus — APPROVED** [review]
/lgtm

File: pkg/csi/cinder/controllerserver.go [hunk 2]
```diff
@@ -530,14 +530,14 @@ func (cs *controllerServer) ValidateVolumeCapabilities(ctx context.Context, req
...
+			return nil, status.Errorf(codes.NotFound, "ValidateVolumeCapabilities Volume %s not found", volumeID)
 		}
-		return nil, status.Errorf(codes.Internal, "ValidateVolumeCapabiltites %v", err)
+		return nil, status.Errorf(codes.Internal, "ValidateVolumeCapabilities %v", err)
 	}
 
 	for _, cap := range reqVolCap {
 		if cap.GetAccessMode().GetMode() != cs.Driver.vcap[0].Mode {
-			return &csi.ValidateVolumeCapabilitiesResponse{Message: "Requested Volume Capabilty not supported"}, nil
+			return &csi.ValidateVolumeCapabilitiesResponse{Message: "Requested Volume Capability not supported"}, nil
```
[the remark this answers is not in the document]
Comment (kayrus) [reply pkg/csi/cinder/controllerserver.go:540]: agree

**Pull request merged**